cmd/kubectl-gadget/networkpolicyadvisor.go

```diff
@@ -169,7 +182,21 @@ func runNetworkPolicyReport(cmd *cobra.Command, args []string) error {
 	}
 
 	advisor.GeneratePolicies()
-	fmt.Printf(advisor.FormatPolicies())
 
-	return err
+	w, closure, err := newWriter(outputFileName)
+	if err != nil {
+		contextLogger.Fatalf("Error creating file %q: %q", outputFileName, err)
```

Comment:
Does it look better with the following?

```suggestion
		contextLogger.Fatalf("Error creating file %q: %s", outputFileName, err)
```

Ditto in other contextLogger.Fatalf calls.